mlxbf-bootctl.c

```diff
@@ -388,12 +471,19 @@ void show_status(void)
     (boot_bus_width & EXT_CSD_BOOT_BUS_WIDTH_RESET_MASK)? "FALSE" : "TRUE");
 
   // Display the watchdog value
-  int watchdog = get_watchdog();
+  uint8_t watchdog_mode = get_watchdog_mode();
+  const char *watchdog_mode_str = get_watchdog_mode_str();
+  uint16_t watchdog_interval = get_watchdog_interval();
+  printf("boot watchdog mode: %s\n", watchdog_mode_str);
+  printf("boot watchdog interval: %d\n", watchdog_interval);
```

Comment:
This commit is for Jan release, but merging it now might cause potential regression issue for Dec release. Just a question, is there a way to differentiate old/new version,  so we could keep the same output for Dec release?